### DISCUSSION TITLE
Onboarding PR against non-default branch

### DISCUSSION THREAD

**Discussion by Andersbiha** [renovatebot/renovate#26321] [Request Help] [answered]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

37.99.1

### If you're self-hosting Renovate, select which platform you are using.

Gitea or Forgejo

### What is your question?

Is it possible to configure Renovate to raise the onboarding PR against another branch (dev)?

### Logs (if relevant)

<details><summary>Logs</summary>

```

Copy/paste the relevant log(s) here, between the starting and ending backticks

```

</details>

**Comment by HonkingGoose** [comment]
I think the `onboardingBranch` [^1] config option is what you want. **Edit:** no this is not what you want, that option only changes the branch name that Renovate will create. It does not change the branch targeted... :upside_down_face: 

Let's try again... You can set your own `baseBranches` [^2] that Renovate will detect and process.

[^1]: [Renovate docs, self-hosted config options `onboardingBranch`](https://docs.renovatebot.com/self-hosted-configuration/#onboardingbranch)
[^2]: [Renovate docs, `baseBranches`](https://docs.renovatebot.com/configuration-options/#basebranches)

**Reply by Andersbiha** [reply]
Unfortunatly none of the above does the trick. The `baseBranches` seem to configure which branches Renovate should run against after onboarding is completed.

I'm trying to get Renovate to raise the Onboarding PR against my dev branch and not do anything further until it detects the config have been merged into main.

**Reply by HonkingGoose** [reply]
I'm out of ideas now. 🙂

I'll let a maintainer respond, maybe they know how/if you can get Renovate to use a different base branch when onboarding.

**Comment by rarkins** [comment, marked as answer]
It's not possible to raise the onboarding PR against a non-default branch. A repo must either have a config in the default branch, or you set requireConfig=optional (no config) and have no onboarding

**Reply by Andersbiha** [reply]
Thanks for the quick reply.
